discord/enums.py

```diff
@@ -654,8 +654,13 @@ class ComponentType(Enum):
 
     action_row = 1
     button = 2
-    select = 3
+    select = 3  # alias for string_select
```

Comment:
Maybe deprecate this naming in favor of `string_select`? A property that returned `ComponentType.string_select` might work.